packages/client/src/api/leaderboardApi.ts

```diff
@@ -0,0 +1,36 @@
+import { defaultHeaders } from 'src/constants/http';
+import { SCORE_FIELD_NAME, TEAM_NAME } from 'src/constants/LeaderboardConsts';
+import {
+  ILeaderboardRequest,
+  IScore,
+  Leader,
+} from 'src/store/leaderboard/types';
+import { fetchApi } from 'src/store/utils';
+
+const prepareLeaderboardRequestData = (
+  score: number,
+  login: string
+): IScore => {
+  return {
+    data: { [SCORE_FIELD_NAME]: score, user: login },
+    teamName: TEAM_NAME,
+    ratingFieldName: SCORE_FIELD_NAME,
+  };
+};
+
+export const addLeader = async (score: number, login = '') => {
+  return await fetchApi<Leader[]>('/leaderboard', {
```

Comment:
```suggestion
export const addLeader = (score: number, login = '') => {
  return fetchApi<Leader[]>('/leaderboard', {
```

Так как мы тут ничего не ждем - это просто функция возвращающая промис, тут не нужен не async не await